modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/Logging.scala

```diff
@@ -100,6 +103,13 @@ object Logging {
           case NonFatal(_) => ()
         }
 
+      def sendMetrics(loaded: LoaderMessage.ShreddingComplete): F[Unit] =
+        for {
+          metrics <- statsD.sendMetrics(loaded)
+          metricsStr = metrics.map { case StatsD.KVMetric(k, v) => s"$k:$v"}.mkString(", ")
+          _ <- if(metricsStr.nonEmpty) info(s"Metrics: $metricsStr") else Sync[F].unit
```

Comment:
Now that we have the metric with load latency this can never be empty, I need to remove the check